### DISCUSSION TITLE
How to use datetime to do an update

### DISCUSSION THREAD

**Discussion by gleysonabreu** [prisma/prisma#11551] [Q&A] [answered]
I'm trying to do an update many where createdAt has a certain date.
But the date I have in the database is 2022-01-31 14:22:10.715.
I've tried several methods searched on the internet and nothing worked correctly.
So I need it to change in places that have the userId, broomId and createdAt date with 2022-01-31 basically.
Is there any method in the prisma that I still haven't found that helps with this?
Thanks.

`prismaClient.reward.updateMany({
        where: {
          userId,
          broomId,
          createdAt,
        },
       data: { .... }
      }),`

**Comment by niranjangajare** [comment]
Try Using [Filter Conditions](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#contains)

```javascript
prismaClient.reward.updateMany({ 
    where: { 
        userId, 
        broomId, 
        createdAt: { 
            contains: "2022-01-31" 
        },
    }, 
    data: { .... }
})
```

**Reply by gleysonabreu** [reply]
contains doesn't work with DateTime I think.

**Comment by gleysonabreu** [comment, marked as answer]
So I change my dates to UTC+0 and now I use gte and lt inside createdAt, it works fine now.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
